src/jbms/argparse.cpp

```diff
@@ -114,6 +116,42 @@ static string join(string_view sep, Range const &args, Transform &&t = {}, Predi
   return ostr.str();
 }
 
+/**
+ * @brief Structure to hold the terminal width's and height's information.
+ * @a row Terminal row number.
+ * @a col Terminal column number.
+ */
+struct win {
+  int row;
+  int col;
+};
+
+int64_t get_terminal_dimensions(struct win *__w) {
+  /**
+   * @brief Fills the given object with terminal dimensions.
+   * @param __w @a win object.
+   */
+#if defined(_WIN32) || defined(_WIN64) || \
```

Comment:
You don't need to separately check `_WIN64`, since `_WIN32` is always defined even in 64-bit builds:

https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?redirectedfrom=MSDN&view=msvc-160